.github/workflows/merge.yml

```
@@ -0,0 +1,38 @@
+      with:
+        fetch-depth: 0
+    - name: Set Git config
+      run: |
+        git config --local user.name "github-actions[bot]"
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+    - name: Rebase to target branch when needed in order to fast-forward merge
+      run: |
+        git fetch origin ${{ github.event.pull_request.base.ref }}
+        git rebase origin/${{ github.event.pull_request.base.ref }}
+    - name: Push back source branch (head)
+      run: |
```

actionlint

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)